### PR TITLE
Tweaking Security Outfits

### PR DESCRIPTION
**Pull request by alsoandanswer**
- tweak: "Changes dark blue warden uniform to match the rest of the department, for consistency reasons and the fact it did not match with any other security accessories"
  - imageadd: "Added unused tan, blue and navy blue security jackets to the loadout, with some tweaking."
  - imageadd: "Red forensic tech jacket has been removed for being out of place. In lieu of this, similar tan, blue and navy blue variants have been added. The badge has been removed in order to further distance resemblance from the FIB jacket."
  - imageadd: "FIB jacket has also been updated."
  - rscadd: "FIB and forensic technician jackets are now able to open and close like labcoats."
  - balance: "Removed the armor values from cosmetic coats in security lockers."
  - rscadd: "Resprited warden and hos caps. added a corporate version as well, and moved them to an appropriate file, along with the commissar's cap."
  - rscadd: "Finally, added corporate bandanas"

dreammaker_sV1HsMo3vB

dreammaker_662rcH4m7u

dreammaker_0WBBkxpEaU

dreammaker_foOuWfxzep

### CODE REVIEW

**Comment by TheDocOct** [comment]
Feedback thread for the sprite changes, please.

**Comment by alsoandanswer** [comment]
> 
> 
> Feedback thread for the sprite changes, please.

https://forums.aurorastation.org/topic/15150-feedback-thread-security-clothing-tweaks/
ok m9

**Comment by alsoandanswer** [comment]
!review

**Comment by NonQueueingMatt** [comment]
Tans are a no-go for security. We have too many colors already.

**Comment by alsoandanswer** [comment]
You think I could use them for Zavodskoi Sec? Not really too attached to them anyway.

**Comment by alsoandanswer** [comment]
nvm i removed them

**Comment by NonQueueingMatt** [comment]
If we're removing armor values from security coats, the detective's vest should be buffed to match officer vests (if it doesn't already) and also have its coat neutered.

The commissar's shit should just go. I don't even need to explain why.

Lastly, how does the warden's uniform look on a warden? It being darker was not exactly bad as it differentiated wardens from officers. So long as wardens still look recognizably different in armor then it is fine imo.

**Comment by alsoandanswer** [comment]
detective vest has exact same armor values due to being just a subclass of the regular ol' vests, so no changes needed

commissar stuff has been removed and armor values from detective trench coats have been accordingly removed

here's a line up of officer, warden and HOS

dreamseeker_XlDxJ8BItO

File: html/changelogs/wezzy_securitytweaks.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - tweak: "Changes dark blue warden uniform to match the rest of the department, for consistency reasons and the fact it did not match with any other security accessories"
+  - imageadd: "Added unused tan, blue and navy blue security jackets to the loadout, with some tweaking."
```
Comment (NonQueueingMatt) [inline html/changelogs/wezzy_securitytweaks.yml:42]: No tan uniforms.

**Review by NonQueueingMatt — DISMISSED** [review]

File: html/changelogs/wezzy_securitytweaks.yml (hunk 1 above)
Comment (alsoandanswer) [reply html/changelogs/wezzy_securitytweaks.yml:42]: Oops

**Review by JohnWildkins — APPROVED** [review]
nothing like greenchecking a controversial security sprite PR

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**